src/test/kotlin/io/github/rybalkinsd/kohttp/ext/StringAsyncExtTest.kt

```diff
@@ -8,14 +8,14 @@ import kotlin.test.assertEquals
 /**
  * @author sergey
  */
-class AsyncStringExtKtTest {
+class StringAsyncExtTest {
```

Comment:
What is the reason for rename?

Reply:
Tests are better grouped up in file list, because files are placed in alphabetical order